src/main/webapp/app/course/course-scores/course-scores-csv-row-builder.ts

```diff
@@ -1,31 +1,24 @@
 import { ExerciseType } from 'app/entities/exercise.model';
 import { EMAIL_KEY, NAME_KEY, POINTS_KEY, REGISTRATION_NUMBER_KEY, SCORE_KEY, USERNAME_KEY } from 'app/course/course-scores/course-scores.component';
 import { CourseScoresStudentStatistics } from 'app/course/course-scores/course-scores-student-statistics';
-
-/**
- * A function that converts a number into the localized representation for the user.
- */
-type Localizer = (value: number) => string;
+import { CsvDecimalSeparator } from 'app/shared/export/csv-export-modal.component';
 
 export type CourseScoresCsvRow = any;
 
 /**
  * Builds CSV rows for the course scores export.
  */
 export class CourseScoresCsvRowBuilder {
-    private readonly localizer: Localizer;
-    private readonly percentLocalizer: Localizer;
+    private readonly decimalSeparator: CsvDecimalSeparator;
 
     private csvRow = {};
 
     /**
      * Creates a new empty CSV row.
-     * @param localizer The function that should be used to convert numbers into their localized representations.
-     * @param percentLocalizer The function that should be used to convert percentage values into their localized representations.
+     * @param decimalSeparator The separator that should be used for numbers.
      */
-    constructor(localizer: Localizer, percentLocalizer: Localizer) {
-        this.localizer = localizer;
-        this.percentLocalizer = percentLocalizer;
+    constructor(decimalSeparator: CsvDecimalSeparator) {
```

Comment:
Maybe the constructor could get the course (or only its rounding accuracy) so that the rounding can be performed within `setLocalozed`/`setLocalizedPercent` before converting the value to a string.